crowbar_framework/app/assets/javascripts/application.js

```diff
@@ -95,9 +78,35 @@ jQuery(document).ready(function($) {
     ]
   });
 
-  setBlockUI('submit', $('[data-blockui]'));
-  setBlockUI('click', $('body.backups').find('[data-blockui]'));
-  setBlockUI('click', $('body.installer-upgrades').find('[data-blockui]'));
+  $('[data-blockui]').live('submit', function(event) {
+    $.blockUI({
+      css: {
+        border: 'none',
+        padding: '15px',
+        backgroundColor: '#000',
+        '-webkit-border-radius': '10px',
+	    '-moz-border-radius': '10px',
```

Comment:
wrong indentation (or if you want to keep it that way, then it's wrong in the other call)